admin/code/CMSBatchActionHandler.php

```diff
@@ -179,7 +179,11 @@ protected function actionByName($name) {
 		// Find the action handler
 		$actions = $this->batchActions();
 		if(!isset($actions[$name]['class'])) {
-			throw new InvalidArgumentException("Invalid batch action: {$name}");
+			$action = $actions[$name];
+
+			throw new InvalidArgumentException(
+				require __DIR__ . '/../templates/Exceptions/InvalidBatchActionException.php'
```

Comment:
I was a little unsure of where to put this, and I think your suggestion is great. how about `messages`?